docs/_posts/2018-05-01-jekyll-3-8-1-released.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Jekyll 3.8.1 Released'
+date: 2018-05-01 16:00:00 +0600
+author: pathawks
+version: 3.8.1
+categories: [release]
+---
+
+Happy May Day :tada:
+
+The Jekyll team is today happy to announce the release of `v3.8.1`, which fixes
```

Comment:
`..is today happy..`?

Reply:
"...is happy to today announce..." ?

Maybe just drop "today?"

Reply:
yeah.. lets just drop "today" :smiley: